[2025-02-28T08:17:04.867+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_churn_ml_pipeline.data_validation_task manual__2025-02-28T08:16:44.276454+00:00 [queued]>
[2025-02-28T08:17:04.887+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_churn_ml_pipeline.data_validation_task manual__2025-02-28T08:16:44.276454+00:00 [queued]>
[2025-02-28T08:17:04.888+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-02-28T08:17:04.917+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): data_validation_task> on 2025-02-28 08:16:44.276454+00:00
[2025-02-28T08:17:04.924+0000] {standard_task_runner.py:57} INFO - Started process 6684 to run task
[2025-02-28T08:17:04.930+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_churn_ml_pipeline', 'data_validation_task', 'manual__2025-02-28T08:16:44.276454+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/customer_churn.py', '--cfg-path', '/tmp/tmpj1_p18dr']
[2025-02-28T08:17:04.935+0000] {standard_task_runner.py:85} INFO - Job 103: Subtask data_validation_task
[2025-02-28T08:17:04.978+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-28T08:17:05.066+0000] {task_command.py:416} INFO - Running <TaskInstance: customer_churn_ml_pipeline.data_validation_task manual__2025-02-28T08:16:44.276454+00:00 [running]> on host 4470298645d8
[2025-02-28T08:17:05.326+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='customer_churn_ml_pipeline' AIRFLOW_CTX_TASK_ID='data_validation_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T08:16:44.276454+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-28T08:16:44.276454+00:00'
[2025-02-28T08:17:05.329+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-02-28T08:17:05.330+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n\t\t\tcd /opt/***\n                        dvc pull /opt/***/data/raw\n                        python /opt/***/executables/data_validation.py && \t\t\tgit rm -r --cached \'reports\'\n\t\t\tgit commit -m "stop tracking reports"\n                        dvc add /opt/***/reports/csv_validation_report.csv &&                         git add /opt/***/reports/csv_validation_report.csv.dvc &&                         git commit -m "Updated validation" &&                         dvc push && git push\n                    ']
[2025-02-28T08:17:05.344+0000] {subprocess.py:86} INFO - Output:
[2025-02-28T08:17:09.826+0000] {subprocess.py:93} INFO - Everything is up to date.
[2025-02-28T08:17:11.939+0000] {subprocess.py:93} INFO - rm 'reports/csv_validation_report.csv.dvc'
[2025-02-28T08:17:11.948+0000] {subprocess.py:93} INFO - [main af3e188] stop tracking reports
[2025-02-28T08:17:11.949+0000] {subprocess.py:93} INFO -  1 file changed, 5 deletions(-)
[2025-02-28T08:17:11.949+0000] {subprocess.py:93} INFO -  delete mode 100644 reports/csv_validation_report.csv.dvc
[2025-02-28T08:17:13.676+0000] {subprocess.py:93} INFO - 
[2025-02-28T08:17:13.676+0000] {subprocess.py:93} INFO - To track the changes with git, run:
[2025-02-28T08:17:13.677+0000] {subprocess.py:93} INFO - 
[2025-02-28T08:17:13.688+0000] {subprocess.py:93} INFO - 	git add reports/csv_validation_report.csv.dvc
[2025-02-28T08:17:13.688+0000] {subprocess.py:93} INFO - 
[2025-02-28T08:17:13.688+0000] {subprocess.py:93} INFO - To enable auto staging, run:
[2025-02-28T08:17:13.688+0000] {subprocess.py:93} INFO - 
[2025-02-28T08:17:13.689+0000] {subprocess.py:93} INFO - 	dvc config core.autostage true
[2025-02-28T08:17:13.973+0000] {subprocess.py:93} INFO - [main a11ddcf] Updated validation
[2025-02-28T08:17:13.973+0000] {subprocess.py:93} INFO -  1 file changed, 5 insertions(+)
[2025-02-28T08:17:13.974+0000] {subprocess.py:93} INFO -  create mode 100644 reports/csv_validation_report.csv.dvc
[2025-02-28T08:17:19.041+0000] {subprocess.py:93} INFO - Everything is up to date.
[2025-02-28T08:17:20.900+0000] {subprocess.py:93} INFO - To https://github.com/anniemathew-trial/customer_churn_pipeline.git
[2025-02-28T08:17:20.901+0000] {subprocess.py:93} INFO -    30f6e6c..a11ddcf  main -> main
[2025-02-28T08:17:20.909+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-28T08:17:20.981+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=customer_churn_ml_pipeline, task_id=data_validation_task, execution_date=20250228T081644, start_date=20250228T081704, end_date=20250228T081720
[2025-02-28T08:17:21.053+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-28T08:17:21.129+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
